### PR TITLE
Fix homepage view bugs

### PR DESCRIPTION
**Pull request by aldenlamp**
## Overview

There are two goals of this PR: the first is to get the homepage's collection views fixed to a presentable state, the second is to clean up the home page code and remove unused code.

Side Note: I also added the new graphql schema for future networking purposes (not connected to the project yet). 

### Views
For the general home page, I fixed the collection view reloading so that it properly loads the views when returning a networking call. I also updated the shadow code for the activity cells to make it match the Figma more accurately. 

### Code
Many of the extra collection view sections that we were not using have been deleted from the home page keeping only those that we need (Today's classes, Your Activities, and Gyms). 

## Test Coverage

There was no significant testing other than checking that it looks more similar to the Figma along with checking that the views load properly. 

## Next Steps  

The views of the homepage still need to be updated. Specifically, the classes need to be redesigned and the gyms need to display capacity.

## Screenshots (delete if not applicable)

<details>

  <summary>Home page appearance</summary>

Simulator Screen Shot - iPhone 14 - 2023-04-26 at 15 27 56

  
  

</details>

### CODE REVIEW

File: Uplift/Controllers/HomeViewController.swift [hunk 1]
```diff
@@ -157,13 +138,16 @@ extension HomeViewController {
...
+        
         collectionView.register(HomeSectionHeaderView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: HomeSectionHeaderView.identifier)
-        collectionView.register(CheckInsListCell.self, forCellWithReuseIdentifier: Constants.checkInsListCellIdentifier)
-        collectionView.register(GymsListCell.self, forCellWithReuseIdentifier: Constants.gymsListCellIdentifier)
-        collectionView.register(ActivitiesListCell.self, forCellWithReuseIdentifier: Constants.activitiesListCellIdentifier)
+        
         collectionView.register(TodaysClassesListCell.self, forCellWithReuseIdentifier: Constants.todaysClassesListCellIdentifier)
         collectionView.register(TodaysClassesEmptyCell.self, forCellWithReuseIdentifier: Constants.todaysClassesEmptyCellIdentifier)
-        collectionView.register(LookingForListCell.self, forCellWithReuseIdentifier: Constants.lookingForListCellIdentifier)
+
```
Comment (emarc0314) [inline Uplift/Controllers/HomeViewController.swift:146]: remove space

File: Uplift/Controllers/HomeViewController.swift [hunk 2]
```diff
@@ -157,13 +138,16 @@ extension HomeViewController {
...
-        collectionView.register(CheckInsListCell.self, forCellWithReuseIdentifier: Constants.checkInsListCellIdentifier)
-        collectionView.register(GymsListCell.self, forCellWithReuseIdentifier: Constants.gymsListCellIdentifier)
-        collectionView.register(ActivitiesListCell.self, forCellWithReuseIdentifier: Constants.activitiesListCellIdentifier)
+        
         collectionView.register(TodaysClassesListCell.self, forCellWithReuseIdentifier: Constants.todaysClassesListCellIdentifier)
         collectionView.register(TodaysClassesEmptyCell.self, forCellWithReuseIdentifier: Constants.todaysClassesEmptyCellIdentifier)
-        collectionView.register(LookingForListCell.self, forCellWithReuseIdentifier: Constants.lookingForListCellIdentifier)
+        
+        collectionView.register(ActivitiesListCell.self, forCellWithReuseIdentifier: Constants.activitiesListCellIdentifier)
+
```
Comment (emarc0314) [inline Uplift/Controllers/HomeViewController.swift:148]: remove space

File: Uplift/Views/Home/ActivityListItemCell.swift [hunk 3]
```diff
@@ -23,7 +23,7 @@ class ActivityListItemCell: ListItemCollectionViewCell<Activity> {
         super.init(frame: frame)
 
         contentView.layer.masksToBounds = false
-        contentView.clipsToBounds = true
+        contentView.clipsToBounds = false
```
Comment (emarc0314) [inline Uplift/Views/Home/ActivityListItemCell.swift:26]: can remove contentView.clipsToBounds

File: Uplift/Controllers/HomeViewController.swift [hunk 4]
```diff
@@ -96,19 +86,10 @@ class HomeViewController: UIViewController {
...
-            self.decrementNumPendingNetworkRequests()
-        })
-
-//MARK: Networking for .LookingForCategories
-/*
-        numPendingNetworkRequests += 1
-        NetworkManager.shared.getTags(completion: { tags in
-            self.lookingForCategories = tags
-            self.collectionView.reloadSections(IndexSet(integer: 2))
+            print(self.gymClassInstances)
```
Comment (emarc0314) [inline Uplift/Controllers/HomeViewController.swift:89]: remove print

File: Uplift/Views/Home/ActivityListItemCell.swift [hunk 5]
```diff
@@ -38,15 +38,15 @@ class ActivityListItemCell: ListItemCollectionViewCell<Activity> {
...
+        let imageViewOpacity: Float = 0.15
+        let imageViewshadowRadius: CGFloat = 4
 
         activityImageView.layer.shadowColor = UIColor.black.cgColor
         activityImageView.layer.shadowOpacity = imageViewOpacity
         activityImageView.layer.shadowOffset = CGSize(width: 1, height: 2)
         activityImageView.layer.shadowRadius = imageViewshadowRadius
         activityImageView.contentMode = .scaleAspectFit
-        activityImageView.clipsToBounds = true
+//        activityImageView.clipsToBounds = true
```
Comment (emarc0314) [inline Uplift/Views/Home/ActivityListItemCell.swift:49]: remove comment

**Review by emarc0314 — CHANGES_REQUESTED** [review]
LGTM 🚀 please address comments and will approve

**Review by emarc0314 — APPROVED** [review]

**Review by ReadeP — APPROVED** [review]

**Pull request merged**